com.unity.formats.alembic/Documentation~/import-options-scene.md

```diff
@@ -10,7 +10,7 @@ Use these options to customize the scale factor, handedness, interpolation, and
 | __Interpolate Samples__ | Enable to interpolate animation for Transform, Camera, and Mesh components for which the topology does not change (that is, for which the number of vertices and indices are immutable).<br/>If **Interpolate Samples** is enabled, or velocity data is included in the .abc file, you can pass velocity data to an Alembic shader. |
 | __Import Visibility__ | Enable to import Visibility (i.e. the control of the active state of the objects). |
 | __Import Camera__ | Enable to import Camera data. |
-| __Import Meshes__ | Enable to import Mesh data. |
+| __Import Meshes__ | Enable to import Mesh data.<br />**Note:** If a mesh includes SubD data, the importer discards this data and converts the base mesh to make it compatible with the Unity Editor. |
```

Comment:
My understanding is that the SubD data is converted into something compatible with the editor.
But I'm a little confused by this sentence, when I read it, it feels the data is not used at all and I'm not sure it is clear what is converted.
It would probably clearer to say the data is discarded after we say we do a conversion.

Reply:
You're right, after re-reading, it feels weird, I should reword.
The conclusion of investigations with Thomas (plus some research I made about SubD):
- SubD involves data and process to calculate smoother surfaces (inner mesh) between vertices/edges of a base polygonal mesh (outer mesh).
- Before, the importer was simply not supporting Alembic files containing any SubD data. No possible import, because Unity doesn't support SubD.
- Now, such files can be imported but Unity keeps only the outer mesh and discards anything normally used to calculate the inner mesh.